dev-python/spyder/spyder-5.0.1.ebuild

```diff
@@ -0,0 +1,202 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8,9} )
+# The warning that this is wrong is a false positive
+# Spyder has setuptools in install_requires
+DISTUTILS_USE_SETUPTOOLS=rdepend
+
+inherit optfeature xdg distutils-r1
+
+# Commit of documentation to fetch
+DOCS_PV="588cdf55cceffea7a3f31d98c0720bfe7df34b72"
+
+DESCRIPTION="The Scientific Python Development Environment"
+HOMEPAGE="
+	https://www.spyder-ide.org/
+	https://github.com/spyder-ide/spyder/
+	https://pypi.org/project/spyder/
+"
+SRC_URI="
+	https://github.com/spyder-ide/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz
+	https://github.com/spyder-ide/${PN}-docs/archive/${DOCS_PV}.tar.gz -> ${PN}-docs-${DOCS_PV}.tar.gz
+"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+# The test suite often hangs or does not work.
+# Technically spyder requires pyqt5<13, which
+# we do not have in ::gentoo any more. Likely
+# this is the reason many of the tests fail
+# or hang. RESTRICTing because IMO it is
+# not worth the several hours I spend every
+# single version bump checking which tests
+# do and do not work. Spyder itself works
+# fine with pyqt5>13.
+RESTRICT="test"
+
+# White space separated deps are expansion of python-language-server[all] dep
```

Comment:
I would recommend something like this:

```
RDEPEND="(general deps)"

# python-language-server[all] deps
RDEPEND+="(...)"
```

This is the next best thing to having comments inline.

Reply:
Thanks, this is much clearer